docs/syntaxes.md

```diff
@@ -0,0 +1,64 @@
+# Syntaxes
+
+Melange supports two syntaxes: OCaml syntax (the default) and Reason syntax.
+Choice of syntax doesn't change the output of the compiler, but Reason syntax is
+designed to be more approachable to JavaScript/TypeScript developers, and it
+supports the use of JSX. Most tooling, such as the [OCaml Platform VS Code
+extension](https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform),
+works well with Reason, but some tools aren't completely aware of it—for
+example, error messages from the compiler still use OCaml syntax.
+
+## Installation
+
+To install Reason syntax support add `reason` to the `depends` section of the
+`<project-name>.opam` file in your project:
```

Comment:
I would expect a link to reasonml docs somewhere here

Reply:
Added links to both OCaml syntax and Reason syntax in the opening paragraph: https://github.com/melange-re/melange-re.github.io/pull/200/commits/5bacbf32a9df043bb0eb2a2fc561b9c20a0f0eba